Makefile

```diff
@@ -35,8 +35,8 @@ RUN := run
 PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
-TESTTIMEOUT  := 15s
-CPUS         := 1
+TESTTIMEOUT  := 1m
+CPUS         := 2,4
```

Comment:
I wonder whether it would be better to use 1,4 here since the =1 case is more different from =4 than =2 is.